torch/_inductor/inductor_prims.py

```diff
@@ -100,6 +105,12 @@ def eager_force_stride(input_tensor: Tensor, stride) -> Tensor:
     lambda a, b, c: (a * b) + c,
     doc="Fused multiply add: fma(a, b, c) -> (a * b) + c without rounding after the multiplication",
 )
+online_softmax = make_prim(
```

Comment:
I wouldn't call this `online_softmax` - I'd call it like `compute_softmax_statistics` or something. it's not actually computing softmax.